### DISCUSSION TITLE
Can't see "authentication token" config

### DISCUSSION THREAD

**Discussion by Alwaysin** [FreshRSS/FreshRSS#5619] [Q&A] [answered]
**Describe the bug**
The "authentication token" config part is not shown on my instance like it is on the screenshots here:

https://github.com/FreshRSS/FreshRSS/blob/edge/docs/en/users/09_refreshing_feeds.md

**To Reproduce**
1. Go to 'Authentication'

![image](https://github.com/FreshRSS/FreshRSS/assets/7189936/1813a2e5-a194-410c-8599-dec7bb3e9d51)

I'm on up-to-date edge.

Nothing of note in the logs, probably a configuration problem on my side?

**Comment by Alkarex** [comment, marked as answer]
Hello,
Check in the *profile management* instead, as this parameter is user-dependant.

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/865db238-313a-4ac0-9ef4-1ba464f2909a)

**Reply by Alwaysin** [reply]
Thanks, I knew I had seen it already but couldn't find it back and I thought it had disappeared. The screenshot is misleading on the doc, it really was moved then and should probably be updated accordingly!

**Reply by Alkarex** [reply]
PR welcome if you spot any possible improvement in the docs :-)
